docs/source/installation.rst

```diff
@@ -4,20 +4,66 @@ Installation
+
+If you don't already have a proper version of Java installed, see `OpenJDK <https://openjdk.java.net/install/>`_ or `Oracle Java <https://www.oracle.com/java/technologies/javase-downloads.html>`_ for download and install instructions for your operating systems.
+
+
+JDBC Driver
 ^^^^^^
+No JDBC drivers are included with the jar-distribution of SchemaSpy.
 
-Download instructions for all operating systems: `https://java.com/en/download/manual.jsp <https://java.com/en/download/manual.jsp>`_
+In some case, a JDBC driver may already exist on your local system if your database happens to come with drivers. Otherwise, you will need to download one yourself.
 
-Optional
---------
+If downloading a driver, you can usually find an approriate driver by searching the internet for "[name of your database] JDBC driver".
 
-Since version 6.1.0 Graphviz is no longer required. There is an embedded viz.js that can be used by adding command line argument ``vizjs``
+Verify the driver you download is compatible with the version of database management system / engine that you are using. For instance, if you're using PostgreSQL 13 the JDBC driver will need to support PostgreSQL 13.
 
-Graphviz
+Here is a list of where you might find drivers for common database management systems:
+
+* `DB2 <https://www.ibm.com/support/pages/db2-jdbc-driver-versions-and-downloads>`_
+* `Firebird <https://firebirdsql.org/en/jdbc-driver/>`_
+* `Impala <https://impala.apache.org/docs/build/html/topics/impala_jdbc.html>`_
+* `MySQL <https://www.mysql.com/products/connector/>`_
+* `MariaDB <https://downloads.mariadb.org/connector-java/>`_
+* `Netezza <https://www.ibm.com/support/knowledgecenter/SSULQD_7.2.1/com.ibm.nz.datacon.doc/c_datacon_installing_configuring_jdbc.html>`_
+* `Oracle <https://www.oracle.com/database/technologies/appdev/jdbc-downloads.html>`_
+* `PostgreSQL <https://jdbc.postgresql.org/download.html>`_
+* `Redshift <https://docs.aws.amazon.com/redshift/latest/mgmt/configuring-connections.html>`_
+* `SQLite <https://github.com/xerial/sqlite-jdbc>`_
+* `SQL Server <https://docs.microsoft.com/en-us/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server>`_
+* `Sybase <http://infocenter.sybase.com/help/index.jsp?topic=/com.sybase.help.sqlanywhere.12.0.1/dbprogramming/jconnect-using-jdbxextra.html>`_
+* `Teradata <https://downloads.teradata.com/download/connectivity/jdbc-driver>`_
+
+You will need to tell SchemaSpy where to find the JDBC driver you require. For example, if you downloaded the Postgres JDBC file called postgresql-42.2.19.jar to the current directory the command would include the -dp parameter, like ``-dp postgresql-42.2.19.jar``. 
+
+If your JDBC .jar file is in a different directory, then the -dp parameter would need to specify the directory path to the file, like ``-dp /opt/some-directory/your-jdbc-driver-name.jar``. 
+
+See :ref:`_commandline` for more information and advanced cases, such if you need to load multiple driver files with ``-loadjars``.
```

Comment:
```
:ref:`_commandline`  

 should be without the leading "_"
```

Reply:
Documentation needs to be updated in regards to `-dp` it will traverse the path and pick upp alla folders and jars.
So the `-loadjars` is sort of deprecated. There's a single nail left in the coffin for it. It's best to specify the file but for drivers with multiple jars. It should be possible to just specify the folder which contains the jars.

Reply:
- Removed the _ in the ref.
- Removed the part about -loadjars.